modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/persistence/PersistentPageMemory.java

```diff
@@ -1291,6 +1305,28 @@ public Set<FullPageId> dirtyPages() {
         return res;
     }
 
+    /**
+     * Checkpoint urgency status enum.
+     */
+    public enum CheckpointUrgency {
+        /**
+         * Signifies that there's enough free space in page memory and there's no need to trigger checkpoint.
```

Comment:
What does “enough” mean? Please expand on this concept.

Reply:
Fixed